<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindowForm</class>
 <widget class="QWidget" name="AddWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="central_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="background_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(216, 216, 216);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="content_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QPushButton" name="submit_product_button">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>160</y>
              <width>111</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Agregar Producto</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="product_name">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>60</y>
              <width>191</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="unidad_medida">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>110</y>
              <width>191</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>30</y>
              <width>181</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>90</y>
              <width>191</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Unidad de medida:</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
